### PR TITLE
Add linux/zypper

### PR DESCRIPTION
**Pull request by kuanyui**
For dear SUSE & openSUSE users.

### CODE REVIEW

**Comment by igorshubovych** [comment]
LGTM

File: pages/linux/zypper.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# zypper
+
+> SUSE & openSUSE package management utility
+
+- Synchronize list of packages and versions available.
+
+`zypper refresh`
+
+- install a new package
```
Comment (felixonmars) [inline pages/linux/zypper.md]: Please capitalize the first letter of each sentence (e.g. "Install" here, and "Remove" on next item).

Comment (kuanyui) [inline pages/linux/zypper.md]: Thank you! Now fixed.

**Comment by felixonmars** [comment]
:+1: Thanks!

**Comment by felixonmars** [comment]
Ah, my bad... Please squash the commits next time :)

**Comment by kuanyui** [comment]
>>Please squash the commits next time :)
>
Ok. I got it.

**Pull request merged**